crates/stackable-operator/CHANGELOG.md

```diff
@@ -1,15 +1,20 @@
 # Changelog
 
+## [Unreleased]
```

Comment:
Please have another look at the diff, there is still something unwanted going on here ;)

Reply:
dammit! [fixed](https://github.com/stackabletech/operator-rs/pull/915/commits/e2f7f7f144229f5f50230343ee0157f290788dee)